### ISSUE TITLE
Fix form controls in header file

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9523]
The `input` tag of the search form needs an accepted `type` attribute: `image` or `submit`. The current type, `search`, is valid HTML but not valid for WCAG.

**Error:** This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

**Code:** H32 #WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2

**Location:** #navbarSupportedContent > div:nth-child(2) > form

```html
<form action="/search/" class="site-header__search form-inline"><input class="site-h...</form>
```

**Comment by allantinashemuzeya** [comment]
Hi @atsansone  how are you ? Can I do this or there is something blocking it?

**Comment by atsansone** [comment]
Thank you for asking! I'm awaiting further a11y testing before continuing with this issue. My apologies if you are waiting for this to be resolved.

**Comment by atsansone** [comment]
Superceded by GAR testing efforts. Closing.
